pep-0681.rst

```diff
@@ -621,9 +622,9 @@ that they should always supply assigned values.
 
 The attrs library supports a bool parameter ``cmp`` that is equivalent
 to setting both ``eq`` and ``order`` to True. We chose not to support
-a ``cmp`` parameter, since it only applies to attrs. Attrs users
-should use the dataclass-standard ``eq`` and ``order`` parameter names
-instead.
+a ``cmp`` parameter, since it only applies to attrs. Users can work
+around this limitation by using the dataclass-standard ``eq`` and
+``order`` parameter names instead.
```

Comment:
I removed "dataclass-standard" from this paragraph, it is the only place the phrase is used.
```suggestion
a ``cmp`` parameter, since it only applies to attrs. Users can emulate
the ``cmp`` behaviour by using the ``eq`` and ``order`` parameter names
instead.
```